.github/workflows/ca-clone-sequential-test.yml

```diff
@@ -1429,6 +1429,774 @@ jobs:
 
           diff expected actual
 
+      ####################################################################################################
+      # Switch cert request ID generator to newLegacy and verify if serials
+      # have gaps when range is updated
+      #
+      # It should work like the legacy but with correct range.
+      - name: Switch primary to newLegacy
+        run: |
+          # Update CS.cfg
```

Comment:
As part of the official migration process we probably should stop the server first to prevent concurrent modification.